docs/strict-syntax.md

```diff
@@ -9,7 +9,13 @@ If you are still using DSL1, see {ref}`dsl1-page` to learn how to migrate your N
 :::
 
 :::{versionadded} 25.04.0
-The strict syntax can be enabled in Nextflow by setting the environment variable `NXF_SYNTAX_PARSER=v2`.
+To enable strict syntax, set the `NXF_SYNTAX_PARSER` environment variable to `v2`:
+
+```bash
+export NXF_SYNTAX_PARSER=v2
+```
+
+Without this environment variable set, Nextflow will not enforce strict syntax rules. Add this to your shell profile (e.g., `~/.bashrc`, `~/.zshrc`) or set it before you run Nextflow commands.
```

Comment:
I'm not sure we should recommend setting this flag globally, although it may be sensible for some. I suspect most people will need to toggle it at a pipeline-level for the time being.